app/workers/education_form/forms/base.rb

```diff
@@ -64,14 +64,25 @@ def initialize(app)
       @text = format unless self.class == Base
     end
 
+    # rubocop:disable Style/AsciiComments
+
+    # @note
+    #   The input fixtures in {spec/fixtures/education_benefits_claims/**/*.json contain
+    #   Windows-1252 encoding "double right-single-quotation-mark", (’) Unicode %u2019
+    #   but .spl file (expected output) contains ASCII apostrophe ('), code %27.
+    #
+    #   Workaround is to sub the ASCII apostrophe, though other non-UTF-8 chars might break specs
+    #
     # Convert the JSON/OStruct document into the text format that we submit to the backend
+    # rubocop:enable Style/AsciiComments
     def format
       @applicant = @form
       # the spool file has a requirement that lines be 80 bytes (not characters), and since they
       # use windows-style newlines, that leaves us with a width of 78
       wrapped = word_wrap(parse_with_template_path(@record.form_type), line_width: 78)
+      wrapped = wrapped.gsub('’', "'")
```

Comment:
This can go in the `en.yml` file under the transliteration key right?

I hesitate to simply make the spec pass -- if there's a right single quote, there can be a left. Same for double quotes.

I *really* wish we had a better idea if this work was even necessary for the output file.

Reply:
> I hesitate to simply make the spec pass -- if there's a right single quote, there can be a left. Same for double quotes.

Agreed. Is there a way to just default unknown transliterations to some default character, like `?` for example, or `□`

Reply:
It defaults them to `?` automatically, but that fails the spec

Reply:
I tried using `en.yml` but it didn't fix the specs https://dsva.slack.com/archives/CQHBJ5U06/p1593465483117100?thread_ts=1593450526.102700&cid=CQHBJ5U06

Reply:
[Looks like it defaults to `?` for replacements](https://github.com/rails/rails/blob/fbe2433be6e052a1acac63c7faf287c52ed3c5ba/activesupport/lib/active_support/inflector/transliterate.rb#L62).

Don't bother with the gsub imo, just allow the default `?` to occur.

Reply:
If this approach is okay, I think we could be a bit more aggressive with quote characters.
I didn't link the other ticket because I wasn't sure if this was ok with everybody.  I'll link it

Reply:
> [Looks like it defaults to `?` for replacements](https://github.com/rails/rails/blob/fbe2433be6e052a1acac63c7faf287c52ed3c5ba/activesupport/lib/active_support/inflector/transliterate.rb#L62).
> 
> Don't bother with the gsub imo, just allow the default `?` to occur.

Update the fixtures to show `?`s instead of apostrophes?

Reply:
I'm happy either way - trying our best to replace quotes or just letting them default to `?`.